stable/hoard/templates/deployment.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: apps/v1
```

Comment:
We should throw if storage.type is "s3" or "gcs" but no credentialsSecret is given.